### DISCUSSION TITLE
Preventing a projector from recording a screen

### DISCUSSION THREAD

**Discussion by LayersOfAbstraction** [rxhanson/Rectangle#1166] [Q&A]
I finally got my project working and wanted to show the audience how it works and why it's helpful. 

Can Rectangle allow me to have many Macbook Air "stages" or a single stage where I screencast an IDE and render my project to a browser while the other screen has the cheat sheet that I hide from everyone?

If so your software would save me from purchasing an extra laptop! Would rather purchase software for that! 😊

**Comment by rxhanson** [comment]
Congrats on getting your project working. It sounds like you just want to make sure that your displays are not mirrored in system settings?
